doc/userguide/install.rst

```diff
@@ -127,18 +127,19 @@ To use it::
 Debian
 ^^^^^^
 
-In Debian 9 (Stretch) do::
+In Debian 9 (stretch) and later do::
 
-    apt-get install suricata
+    sudo apt-get install suricata
 
-In Debian Jessie Suricata is out of date, but an updated version is in Debian Backports.
+In Debian stable Suricata is usually not the latest version, but an updated version is available from Debian backports.
```

Comment:
nit: should we consider adding a comma after "stable" to clarify its not about Suricata?

Reply:
Agreed. See next PR.